Otel doc fixes (#10150)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sources/otel-metrics-source.md

unframed_requests | No | Boolean | Enables requests not framed using the gRPC wire protocol.
thread_count | No | Integer | The number of threads to keep in the `ScheduledThreadPool`. Default value is `200`.
max_connection_count | No | Integer | The maximum allowed number of open connections. Default value is `500`.

> output_format | String | Indicates the output format of the events to be generated. Allowed values are `otel` or `opensearch`. Default value is `opensearch`. |

Reviewer: ```suggestion
| `output_format` | String | Specifies the output format of the generated events. Valid values are `otel` or `opensearch`. Default is `opensearch`. |
```

awsRegion | Conditionally | String | Represents the AWS Region used by ACM or Amazon S3. Required if `useAcmCertForSSL` is set to `true` or `sslKeyCertChainFile` and `sslKeyFile` is the Amazon S3 path.
authentication | No | Object | An authentication configuration. By default, an unauthenticated server is created for the pipeline. This uses pluggable authentication for HTTPS. To use basic authentication, define the `http_basic` plugin with a `username` and `password`. To provide customer authentication, use or create a plugin that implements [GrpcAuthenticationProvider](https://github.com/opensearch-project/data-prepper/blob/1.2.0/data-prepper-plugins/armeria-common/src/main/java/com/amazon/dataprepper/armeria/authentication/GrpcAuthenticationProvider.java).

## Usage

> To get started, create a `pipeline.yaml` file and add `otel_metrics_source` as the source:

Reviewer: ```suggestion
To use the `otel-metrics` source, create the following `pipeline.yaml` file with `otel_metrics_source` as the source:
```
